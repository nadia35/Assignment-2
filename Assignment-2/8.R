averageLastThree <- function(values){
  
  mean_nums <- mean(values[(length(values) - 2): length(values)])
  return(mean_nums)
  
}
