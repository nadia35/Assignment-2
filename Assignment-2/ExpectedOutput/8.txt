[1] 8
[1] 1
[1] 0.3333333
[1] 486.4737
